### DISCUSSION TITLE
Set websocket ping-pong timeout when launching with Gunicorn

### DISCUSSION THREAD

**Discussion by wenleix** [Kludex/uvicorn#2005] [Q&A]
Hi, 

I noted for Uvicorn, we can customize websocket ping-pong timeout with `--ws-ping-interval` and `--ws-ping-timeout` (https://www.uvicorn.org/deployment/#running-from-the-command-line)  . I am wondering if we can set it when the Uvicorn workers are launched with Gunicorn? 

Thanks!

**Comment by Zhaoxun** [comment]
I don't think it is the right place to ask gunicorn features, since they are not the authors of gunicorn. But uvicorn provides multiple workers option `--workers INT`. Wonder if it can replace the functionality of 'Uvicorn workers launched by Gunicorn'
